protocol/app/msgs/unsupported_msgs.go

```diff
@@ -15,8 +14,13 @@ var (
 		"/cosmos.gov.v1beta1.MsgSubmitProposalResponse": nil,
 
 		// ICA Controller messages - these are not used since ICA Controller is disabled.
-		"/ibc.applications.interchain_accounts.controller.v1.MsgSendTx": &icacontrollertypes.MsgSendTx{},
+		"/ibc.applications.interchain_accounts.controller.v1.MsgUpdateParams": &icacontrollertypes.
+			MsgUpdateParams{},
+		"/ibc.applications.interchain_accounts.controller.v1.MsgUpdateParamsResponse": nil,
+		"/ibc.applications.interchain_accounts.controller.v1.MsgSendTx":               &icacontrollertypes.MsgSendTx{},
+		"/ibc.applications.interchain_accounts.controller.v1.MsgSendTxResponse":       nil,
 		"/ibc.applications.interchain_accounts.controller.v1.MsgRegisterInterchainAccount": &icacontrollertypes.
 			MsgRegisterInterchainAccount{},
+		"/ibc.applications.interchain_accounts.controller.v1.MsgRegisterInterchainAccountResponse": nil,
```

Comment:
nit: can we update these to be alphabetic?

Reply:
Done